openfga/v2beta1/openfga_service.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package openfga.v2beta1;
+
+import "google/protobuf/struct.proto";
+import "openfga/v2beta1/openfga.proto";
+
+service OpenFGAService {
+    rpc Check(CheckRequest) returns (CheckResponse);
+    rpc ListObjects(ListObjectsRequest) returns (ListObjectsResponse);
+
+    rpc DeleteRelationships(DeleteRelationshipsRequest) returns (DeleteRelationshipsResponse);
+    rpc WriteRelationships(WriteRelationshipsRequest) returns (WriteRelationshipsResponse);
```

Comment:
Relationships has a specific connotation - this is just writing the tuples, but it does not mean that a relationship will exist

Reply:
Will rename to `WriteRelationshipTuples` to coincide with the entity being written `RelationTuple`.